app/livechat/client/messageBoxActions.js

```diff
@@ -0,0 +1,59 @@
+import { Meteor } from 'meteor/meteor';
+import { Session } from 'meteor/session';
+import { ReactiveVar } from 'meteor/reactive-var';
+import { Tracker } from 'meteor/tracker';
+
+import { messageBox } from '../../ui-utils/client';
+import { t } from '../../utils/client';
+import { settings } from '../../settings/client';
+import { Rooms } from '../../models';
+import { ScrShareDialog } from './views/app/ui-screensharing/client';
+import { screenSharingStreamer } from './lib/stream/screenSharingStream';
+
+const sessions = new ReactiveVar([]);
+
+messageBox.actions.add('Screen_Sharing', 'Request_Screen_Sharing', {
+	id: 'request-screen-sharing',
+	icon: 'video',
+	condition: () => {
+		const rid = Session.get('openedRoom');
+		const room = Rooms.findOne({
+			_id: rid,
+		});
+
+		return settings.get('Livechat_enabled') && settings.get('Livechat_screen_sharing_enabled') && room && room.t === 'l' && !sessions.get().includes(rid);
+	},
+	action: ({ rid, tmid, messageBox }) => {
+		Meteor.call('livechat:requestScreenSharing', rid);
```

Comment:
No, it will not be so
This UI for starting screensharing will not be available when session is active.
I am controlling the session status is screenSharingManager.ts

If screensharing is active in some room then there will be option to end the session or go to the current session if page is reloaded and there will be no option to request screen sharing there